src/Uno.UI.RuntimeTests/MUX/Microsoft_UI_Xaml_Controls/TabView/TabViewTests.cs

```diff
@@ -0,0 +1,226 @@
+﻿#if !WINDOWS_UWP
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using MUXControlsTestApp.Utilities;
+using System;
+using System.Threading;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media;
+using Common;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Provider;
+using Private.Infrastructure;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+using Symbol = Windows.UI.Xaml.Controls.Symbol;
+
+#if !WINDOWS_UWP
+using SymbolIconSource = Windows.UI.Xaml.Controls.SymbolIconSource;
+#else
+using SymbolIconSource = Microsoft.UI.Xaml.Controls.SymbolIconSource;
+#endif
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+
+	[TestClass]
+	public class TabViewTests
+	{
+
+		[TestMethod]
+		public void VerifyCompactTabWidthVisualStates()
+		{
+			TabView tabView = null;
+			RunOnUIThread.Execute(() =>
+			{
+				tabView = new TabView();
+				TestServices.WindowHelper.WindowContent = tabView;
+
+				tabView.TabItems.Add(CreateTabViewItem("Item 0", Symbol.Add));
+				tabView.TabItems.Add(CreateTabViewItem("Item 1", Symbol.AddFriend));
+				tabView.TabItems.Add(CreateTabViewItem("Item 2"));
+
+				tabView.SelectedIndex = 0;
+				tabView.SelectedItem = tabView.TabItems[0];
+				(tabView.SelectedItem as TabViewItem).IsSelected = true;
+				Verify.AreEqual("Item 0", (tabView.SelectedItem as TabViewItem).Header);
+				//TestServices.WindowHelper.WindowContent.UpdateLayout();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6063676)